core-services/template-deprecation/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+    petr-muller
+reviewers:
+    petr-muller
```

Comment:
My intention behind this to be (for a short while) a bottleneck for these PRs, to understand the volume with which the allowlist will change in day-to-day operation. Any approver from the root OWNERS will still have the power to approve, so it should not be a real probllem